src/Cake.Core/CakeContext.cs

```diff
@@ -60,15 +64,19 @@ public sealed class CakeContext : ICakeContext
             {
                 throw new ArgumentNullException("processRunner");
             }
+            if (tools == null)
```

Comment:
@patriksvensson shouldn't there be a line between closing `}` and the new `if` block?